examples/api/test/material/scrollbar/scrollbar.0_test.dart

```diff
@@ -18,4 +18,24 @@ void main() {
 
     expect(tester.takeException(), isNull);
   }, variant: TargetPlatformVariant.all());
+
+  testWidgets('The scrollbar should be painted when the user scrolls', (WidgetTester tester) async {
+
```

Comment:
Consider removing this extra line

Reply:
😭

Reply:
![image](https://github.com/user-attachments/assets/507a272d-4333-4ea2-9c20-84d129fc280c)

Reply:
Sorry for that again, I removed it in [test: Remove empty line and use pumps](https://github.com/flutter/flutter/pull/151463/commits/627803e735c1ad86a8ce44fc2d0a76dd7077f9da)